src/components/Homepage/FilterSelect.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { SetStateAction, useState } from 'react';
+import styles from './FilterSelect.module.scss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronUp } from '@fortawesome/free-solid-svg-icons';
+
+export default function FilterSelect({ setRegionFilter }: { setRegionFilter: React.Dispatch<SetStateAction<string>> }) {
+  const optionsList = ['Africa', 'Americas', 'Antarctic', 'Asia', 'Europe', 'Oceania'];
+
+  const [showMenu, setShowMenu] = useState<boolean>(false);
+  const [selectedFilter, setSelectedFilter] = useState<string>('Filter by Region');
+
+  return (
+    <div className={styles.filterContainer}>
+      <button type="submit" className={styles.selectButton} onClick={() => setShowMenu((showMenu) => !showMenu)}>
+        <p>{selectedFilter}</p>
+        <FontAwesomeIcon
+          icon={faChevronUp}
+          className={[styles.rotateTransition, showMenu ? styles.rotateIcon : ''].join(' ')}
+        />
+      </button>
+      {showMenu && (
+        <div className={styles.optionsList}>
```

Comment:
Are we going to support accessibility? since we opt to use div instead of select

Reply:
Please take a look at my approach, I added aria attributes to the div but idk if they're correct.

Reply:
I mean the native behavior of select eg: the up and down arrow

Reply:
Got it. Added keydown event listeners for up, down, and escape keys.

Reply:
does this cover enter and space key? to select the value?

Reply:
yes I think for buttons you can press them by default using enter or space key